src/udiskslinuxdriveata.h

```diff
@@ -51,6 +51,10 @@ void            udisks_linux_drive_ata_apply_configuration (UDisksLinuxDriveAta
                                                             UDisksLinuxDevice       *device,
                                                             GVariant                *configuration);
 
+gboolean       get_pm_state                                 (UDisksLinuxDevice      *device, 
```

Comment:
Please add namespace to the function name, i.e. `udisks_linux_drive_ata_get_pm_state`.